packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuSplitButton.tsx

```diff
@@ -0,0 +1,178 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  assign,
+  buttonProperties,
+  getNativeProps,
+  KeyCodes
+} from '../../Utilities';
+import { IContextualMenuItem } from '../../ContextualMenu';
+import {
+  IMenuItemClassNames,
+  getSplitButtonVerticalDividerClassNames
+} from './ContextualMenu.classNames';
+import { ContextualMenuItem } from './ContextualMenuItem';
+
+import { getIsChecked, isItemDisabled } from '../../utilities/contextualMenu/index';
+import { VerticalDivider } from '../../Divider';
+import { IContextualMenuSplitButtonProps } from './ContextualMenuSplitButton.types';
+
+export interface IContextualMenuSplitButtonState { }
+
+export class ContextualMenuSplitButton extends BaseComponent<IContextualMenuSplitButtonProps, IContextualMenuSplitButtonState> {
+  private _splitButton: HTMLDivElement;
+
+  public render(): JSX.Element | null {
+    const {
+      item,
+      classNames,
+      index,
+      focusableElementIndex,
+      totalItemCount,
+      hasCheckmarks,
+      hasIcons,
+      onItemMouseLeave,
+      onItemMouseMove
+    } = this.props;
+
+    return (
+      <div
+        ref={ (splitButton: HTMLDivElement) => this._splitButton = splitButton }
+        role={ 'button' }
+        aria-labelledby={ item.ariaLabel }
+        className={ classNames.splitContainer }
+        aria-disabled={ isItemDisabled(item) }
+        aria-haspopup={ true }
+        aria-describedby={ item.ariaDescription }
+        aria-checked={ item.isChecked || item.checked }
+        aria-posinset={ focusableElementIndex + 1 }
+        aria-setsize={ totalItemCount }
+        onMouseEnter={ this._onItemMouseEnter.bind({ ...item, subMenuProps: null, items: null }) }
```

Comment:
Don't do the bind here, instead in move it to the function call in splitbutton like 
```
    const { onItemMouseEnter, item } = this.props;
    if (onItemMouseEnter) {
      onItemMouseEnter({...item, submenuProps: null}, items: null)}, ev, this._splitButton);
    }